engine/schema/src/com/cloud/upgrade/dao/Upgrade410to420.java

```diff
@@ -1297,7 +1297,7 @@ private void addHostDetailsIndex(Connection conn) {
                     s_logger.debug("Index already exists on host_details - not adding new one");
```

Comment:
@wido the index is not added, if you look at the code above ^^ the if statement to check if index already exists with given key name and it only adds the index if it does not exist otherwise skips.